### ISSUE TITLE
Add legend for fieldset elements on register and login forms

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#5183]
Our accessibility audit recommended that we need to include `<legend>` elements in `<fieldset>` tags.
This is intended to be used by screenreaders to describe what are the options or fields to check or fill. 
Also, this is not intended to be visually available but for screenreaders to parse. When adding the `legend `element it should be `CSS `formated to not to be shown on web browser, i.e., `display:hidden;`

We should change the code on login page from this:

```
<fieldset class="fields">
<div class="username">
<label>
<span class="label">
Username
<span class="required">*</span>
<span class="pkp_screen_reader">
Required
</span>
</span>
[...]
</fieldset>
```

To this

```
<fieldset class="fields">
<legend>Username and Password credentials</legend>
<div class="username">
<label>
<span class="label">
Username
<span class="required">*</span>
<span class="pkp_screen_reader">
Required
</span>
</span>
[...]
</fieldset>
```

Also we need add` <legend>` element in `<fieldset class="consent">` and `<fieldset class="review">` elements on register page that are missing them. The consent and reviewer class need to be adjusted. 

The `<fieldset class="consent">`  should change from this:
```
<fieldset class="consent">
<div class="fields">
<div class="optin optin-privacy">
<label>
<input type="checkbox" name="privacyConsent" value="1">
Yes, I agree to have my data collected and stored according to the <a href="https://uiux.publicknowledgeproject.org/ojs/index.php/publicknowledge/about/privacy" target="_blank">privacy statement</a>.
</label>
</div>
[...]
</fieldset>
```

To this:
```
<fieldset class="consent">
<legend>Options to consent<legend>
<div class="fields">
<div class="optin optin-privacy">
<label>
<input type="checkbox" name="privacyConsent" value="1">
Yes, I agree to have my data collected and stored according to the <a href="https://uiux.publicknowledgeproject.org/ojs/index.php/publicknowledge/about/privacy" target="_blank">privacy statement</a>.
</label>
</div>
[...]
</fieldset>
```

And the `<fieldset class="review">`  should change from this:

```
<fieldset class="reviewer">
<div class="fields">
<div id="reviewerOptinGroup" class="optin">
<label>
<input type="checkbox" name="reviewerGroup[16]" value="1">
Yes, I would like to be contacted with requests to review submissions to this journal.
</label>
																					</div>
[...]
</fieldset>

```
To this:
```
<fieldset class="reviewer">
<legend>Consent to be contacted as reviewer</legend>
<div class="fields">
	<div id="reviewerOptinGroup" class="optin">
	<label>
	<input type="checkbox" name="reviewerGroup[16]" value="1">
	Yes, I would like to be contacted with requests to review submissions to this journal.
	</label>
	</div>


[...]
</fieldset>

```

**Comment by thinkbulecount2** [comment]
Hi @israelcefrin your message got cut off for `<fieldset class="review">` ?

**Comment by israelcefrin** [comment]
Hi @thinkbulecount2 , I've added the missing part, sorry for that.

**Comment by thinkbulecount2** [comment]
No problem, @israelcefrin !!

**Comment by thinkbulecount2** [comment]
PRs:
[OJS](https://github.com/pkp/ojs/pull/2803)
[pkp-lib](https://github.com/pkp/pkp-lib/pull/5985)

@NateWr was wondering if the locale strings I created are okay or if need to amend. Let me know!

**Comment by NateWr** [comment]
Merged! :tada:
